src/Testing/src/build/Microsoft.AspNetCore.Testing.props

```diff
@@ -1,9 +1,13 @@
 ﻿<Project>
   <PropertyGroup>
+    <RepoRoot
+        Condition=" '$(RepoRoot)' == '' OR !HasTrailingSlash('$(RepoRoot)') "><RepoRoot>$([MSBuild]::NormalizeDirectory('$(MSBuildThisFileDirectory)', '..', '..', '..', '..'))</RepoRoot></RepoRoot>
```

Comment:
Is `<RepoRoot><RepoRoot>...</RepoRoot></RepoRoot>` a thing or just a typo?

Reply:
@dougbu I'm also wondering about this

Reply:
It's a typo and a dumb one -- cut and paste error ☹️ Fortunately the outer `Condition` is rarely, if ever `true`. PR incoming&hellip;

Reply:
Fixed in #32680